README.md

```diff
@@ -32,72 +36,90 @@ You can chain multiple AI models together to create a pipeline before hitting th
 - VectorDB integration
 
 ## Requirements
+
 - OpenAI API key
 - Postgres
 - Redis
 
-
 #### OpenShield is a firewall designed for AI models.
 
-
 ### Endpoints
+
 ```
 /openai/v1/models
 /openai/v1/models/:model
 /openai/v1/chat/completions
 ```
 
 ## Demo mode
-We are generating automatically demo data into the database. You can use the demo data to test the application.
 
+We are generating automatically demo data into the database. You can use the demo data to test the application.
 
 ```shell
 cd demo
 cp .env.example .env
+cp ./example_config ./config.yaml/config
```

Comment:
cp ./example_config ./config.yaml

Reply:
Similar to the lib/config.go I made this change because we are expecting a file called "config".  Once we resolve the above, we can modify this accordingly.